packages/gatsby-theme-newrelic/src/components/ResultView.js

```diff
@@ -0,0 +1,128 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  appendClassName,
+  getUrlSanitizer,
+} from '@elastic/react-search-ui-views/es/view-helpers';
+import { isFieldValueWrapper } from '@elastic/react-search-ui-views/es/types/FieldValueWrapper';
+import { css } from '@emotion/core';
+import { rgba } from 'polished';
+
+function getFieldType(result, field, type) {
```

Comment:
Nitpick: would you be willing to update these functions to match the style of our other functions:

```suggestion
const getFieldType = (result, field, type) => {
```